profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.install

```diff
@@ -64,6 +96,21 @@ function nexteuropa_varnish_install() {
 }
 
 /**
+ * Implements hook_uninstall().
+ */
+function nexteuropa_varnish_uninstall() {
+  $vars = [
+    'nexteuropa_varnish_http_targets',
+    'nexteuropa_varnish_tag',
+    'nexteuropa_varnish_request_method',
+  ];
+
+  foreach ($vars as $var) {
+    variable_del($var);
```

Comment:
We delete variables that we have not initiate it during the hook_install.
I think it's not necessary to delete it.